c2corg_ui/static/js/documentservice.js

```diff
@@ -0,0 +1,51 @@
+goog.provide('app.Document');
+
+goog.require('app');
+
+
+/**
+ * This services is used to acces the document properties
+ * within other controllers, for example to set the properties in one 
+ * controller and then read them in another.
+ * @constructor
+ */
+app.Document = function() {
```

Comment:
yep, it could be used for everything. You spoke about our controllers being too document-specific -> now we have a service for that. At least a service which gathers functions that all or most documents have, but then we could also create an outingservice, routeservice etc if needed.